### PR TITLE
[e2e] create screen facade class and refactor objects of page classes

### PR DESCRIPTION
**Pull request by anezboretskiy**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+public abstract class Facade {
+
+    protected ConnectorsView connectorsView = new ConnectorsView();
+    protected MainPage mainPage = new MainPage();
+    protected TopicsList topicsList = new TopicsList();
+    protected TopicView topicView = new TopicView();
+    protected ConnectorsList connectorsList = new ConnectorsList();
+    protected SchemaRegistryList schemaRegistry = new SchemaRegistryList();
+    protected ApiHelper apiHelpers = new ApiHelper();
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java:19]: several apiHelpers?

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import com.provectus.kafka.ui.pages.topic.TopicsList;
+
+public abstract class Facade {
+
+    protected ConnectorsView connectorsView = new ConnectorsView();
+    protected MainPage mainPage = new MainPage();
+    protected TopicsList topicsList = new TopicsList();
+    protected TopicView topicView = new TopicView();
+    protected ConnectorsList connectorsList = new ConnectorsList();
+    protected SchemaRegistryList schemaRegistry = new SchemaRegistryList();
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java:18]: where's schemaForm?

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java (hunk 1 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java]: fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java (hunk 2 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/Facade.java]: added all forms

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 3]
```diff
@@ -77,15 +77,12 @@ public void createConnector() {
     @CaseId(196)
     @Test
     public void updateConnector() {
-        pages.openConnectorsList(CLUSTER_NAME)
-                .waitUntilScreenReady()
-                .openConnector(CONNECTOR_FOR_UPDATE.getName());
-        pages.connectorsView
+        connectorsView.goTo(CLUSTER_NAME, CONNECTOR_FOR_UPDATE.getName())
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:80]: we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 4]
```diff
@@ -94,19 +91,17 @@ public void updateConnector() {
     @CaseId(195)
     @Test
     public void deleteConnector() {
-        pages.openConnectorsList(CLUSTER_NAME)
-                .waitUntilScreenReady()
-                .openConnector(CONNECTOR_FOR_DELETE.getName());
-        pages.connectorsView.clickDeleteButton();
-        pages.openConnectorsList(CLUSTER_NAME);
-        Assertions.assertFalse(pages.connectorsList.isConnectorVisible(CONNECTOR_FOR_DELETE.getName()),"isConnectorVisible()");
+        connectorsView.goTo(CLUSTER_NAME, CONNECTOR_FOR_DELETE.getName())
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:94]: we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 5]
```diff
@@ -45,15 +43,15 @@ public static void beforeAll() {
     @Test
     public void createTopic() {
         Topic topicToCreate = new Topic().setName("new-topic");
-        pages.open()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
-        pages.topicsList.pressCreateNewTopic()
+        topicsList.goTo(CLUSTER_NAME)
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:46]: we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 6]
```diff
@@ -45,15 +43,15 @@ public static void beforeAll() {
...
+                .waitUntilScreenReady()
+                .pressCreateNewTopic()
                 .setTopicName(topicToCreate.getName())
                 .sendData()
                 .waitUntilScreenReady();
-        pages.open()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
-        Assertions.assertTrue(pages.topicsList.isTopicVisible(topicToCreate.getName()),"isTopicVisible");
+        topicsList.goTo(CLUSTER_NAME)
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:52]: we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 7]
```diff
@@ -64,9 +62,9 @@ public void createTopic() {
     @CaseId(197)
     @Test
     public void updateTopic() {
-        pages.openTopicsList(CLUSTER_NAME)
+        topicsList.goTo(CLUSTER_NAME)
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:65]: we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 8]
```diff
@@ -76,16 +74,16 @@ public void updateTopic() {
                 .setMaxMessageBytes(TOPIC_FOR_UPDATE.getMaxMessageBytes())
                 .sendData()
                 .waitUntilScreenReady();
-        pages.openTopicsList(CLUSTER_NAME)
+        topicsList.goTo(CLUSTER_NAME)
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:77]: we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 9]
```diff
@@ -95,14 +93,14 @@ public void updateTopic() {
     @CaseId(207)
     @Test
     public void deleteTopic() {
-        pages.openTopicsList(CLUSTER_NAME)
+        topicsList.goTo(CLUSTER_NAME)
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:96]: we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java (hunk 3 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java]: Done

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java (hunk 4 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java]: Done

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 5 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java]: Done

**Review by VladSenyuta — APPROVED** [review]

**Pull request merged**